### DISCUSSION TITLE
is there's noo other theme that same beautiful as tokynight in lazyvim?

### DISCUSSION THREAD

**Discussion by cStralpt** [LazyVim/LazyVim#1763] [General]
any suggestion would be appreciated. 
![image](https://github.com/LazyVim/LazyVim/assets/95400822/c016f9f9-414a-4dc9-ad73-7986719d4932)

**Comment by amaanq** [comment]
rose-pine or catppuccin are similar

**Reply by cStralpt** [reply]
thank u, it's look nice and fully supported .jsx, the issues for own reference theme lol
